installer/pkg/cmd/gcp_broker.go

```diff
@@ -142,14 +142,14 @@ func addGCPBroker() error {
 	if err != nil {
 		// Clean up the newly generated key if the command failed.
 		cleanupNewKey(brokerSAEmail, key)
-		return fmt.Errorf("error generating configs for GCP broker :: %v", err)
+		return fmt.Errorf("error generating configs for service broker :: %v", err)
```

Comment:
So we are fine with both the "Service Broker" and "service broker"?

Reply:
So I switched all "title-like" strings to Service Broker and all regular outputs or errors to just "service broker"